mesh/api/testdata/test_mesh_graph.expected

```diff
@@ -230,33 +230,12 @@
           "infraData": {
             "ComponentStatuses": {
               "Enabled": true,
-              "Components": [
-                {
-                  "AppLabel": "istio-egressgateway",
-                  "IsCore": false,
-                  "IsProxy": true,
-                  "IsMultiCluster": false,
-                  "Namespace": ""
-                },
-                {
-                  "AppLabel": "istio-ingressgateway",
-                  "IsCore": true,
-                  "IsProxy": true,
-                  "IsMultiCluster": false,
-                  "Namespace": ""
-                },
-                {
-                  "AppLabel": "istiod",
-                  "IsCore": true,
-                  "IsProxy": false,
-                  "IsMultiCluster": false,
-                  "Namespace": ""
-                }
-              ]
+              "Components": null
```

Comment:
This doesn't look right.  I would have expected it to be omitted, or be an empty array.

Reply:
yes, thanks, should be an empty array